documentation/modules/ROOT/pages/operations/openshift.adoc

```diff
@@ -8,56 +8,75 @@
 
 toc::[]
 
-The following describes how to set up the {prodname} connectors for change data capture on Red Hat's https://www.openshift.com/[OpenShift] container platform.
+This procedure is for setting up {prodname} connectors on Red Hat's link:https://www.openshift.com/[OpenShift] container platform. These instructions have been tested with the two most recent releases of OpenShift. 
```

Comment:
Can you add a sentence along the lines of "These instructions should work equally on any other Kubernetes distribution, using the _kubectl_ command".

Reply:
I added: These instructions should also work on any other Kubernetes distribution by using the `kubectl` command.